.github/workflows/docker-build.yml

```diff
@@ -254,74 +224,9 @@ jobs:
           # provenance: false will result in a single manifest for all platforms which makes the image pullable from arm64 machines via the emulation (e.g. Apple Silicon machines)
           provenance: false
 
-  build_components:
```

Comment:
@ogabrielluiz I removed this step. I don't know what these images are used for.

Reply:
These were added to support deployment in DS LF, I'm not sure if they are used at all but I was hoping to create a template in Railway with a decoupled frontend so we could scale the backend with a reverse proxy like NGINX